scripts/token_checker.py

```diff
@@ -26,7 +26,8 @@ def __init__(self):
                             "zksync": "zksync-zksync",
                             "zora": "eth-ethereum",
                             "mantle": "mnt-mantle",
-                            "blast": "blast-blast"
+                            "blast": "blast-blast",
+                            "linea": "eth-ethereum"
```

Comment:
Adding linea support similar to zora.

(Currently linea and zora are still not in the coinpaprika list: https://api.coinpaprika.com/v1/contracts/)